packages/cerebral/typings/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// <reference types="node" />
+// <reference types="function-tree" />
+
+
+declare module 'cerebral/devtools' {
```

Comment:
you just place an d.ts file for each entry point with the same name. typescript compiler itself will resolve it